scripts/scil_flip_volume.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env python
+
+import argparse
+import os.path
+
+import nibabel as nib
+import logging
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_arg_parser():
+
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description='Flip the volume according to the'
```

Comment:
Please put the description in the `_DESCRIPTION_` string, as is done in other scripts.